jib-maven-plugin/README.md

```diff
@@ -221,6 +221,7 @@ Field | Type | Default | Description
 `from` | [`from`](#from-object) | See [`from`](#from-object) | Configures the base image to build your application on top of.
 `container` | [`container`](#container-object) | See [`container`](#container-object) | Configures the container that is run from your image.
 `extraDirectories` | [`extraDirectories`](#extradirectories-object) | See [`extraDirectories`](#extradirectories-object) | Configures the directories used to add arbitrary files to the image.
+`containerizingMode` | `String` | `exploded` | If set to `packaged`, puts the JAR artifact built at `${project.build.directory}/${project.build.finalName}.jar` (the default location where many JAR-buidiling plugins put a JAR registered as a main artifact, such as the Maven JAR Plugin) into the final image. If set to `exploded` (default), containerizes individual `.class` files and resources files.
```

Comment:
nit: for the Maven readme, we've been labeling types like "string" instead of "`String`". Maybe we want to change the whole thing to be more like the gradle readme instead?

Reply:
Ah, I'll fix this. Things in XML are not Java types, so that's probably why we just say "string".